src/app/package.json

```diff
@@ -9,7 +9,7 @@
 		"title": "<%= main['display-name'] %>",
 		"frame": false,
 		"resizable": true,
-		"show": false,
+		"show": true,
```

Comment:
This is set to `false` intentionally and can't be changed.

The app supports various launch parameters which prevent the application window from being shown, and the window does only get shown in the absence of those parameters, after the application has been fully initialized.
- https://github.com/streamlink/streamlink-twitch-gui/blob/v2.2.0/src/app/init/instance-initializers/nwjs/instance-initializer.js#L73-L82
- https://github.com/streamlink/streamlink-twitch-gui/blob/v2.2.0/src/app/init/instance-initializers/nwjs/parameters.js#L32-L47